gensc/README.md

```diff
@@ -1,13 +1,19 @@
 ## Redirect for the Genomics Standards Consortium
 
-w3id.org/gensc
+The Genomic Standards Consortium currently has two w3id namespaces:
+- `gensc`
+- `mixs`
 
-This redirect is used to direct URIs for terms, checklists, packages and other documents related to GSC standards.
+The `mixs` namespace is more actively maintained, but some resources, like [Darwin Core](https://dwc.tdwg.org/) may need to temporarily use the `gensc` namespace, as pointed out by @MattBlissett in
```

Comment:
Probably not necessary to link to PRs (and GitHub usernames) in the README?

The `mixs` namespace is more actively maintained, but some resources like [Darwin Core](https://dwc.tdwg.org/) may need to occasionally use the `gensc` namespace.

Reply:
thanks